front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -141,25 +139,20 @@ const CollapsibleContainer = ({
           <ChevronIcon name="chevron-right" />
         </TitleButton>
       </Title>
-
       <Box
         role={useRegionRole ? 'region' : undefined}
         aria-live="polite"
         id={`collapsed-section-${uuid}`}
         aria-labelledby={`collapse-container-title-${uuid}`}
-        hidden={!isExpanded}
```

Comment:
The animation didn't work with this. From my quick research, I gathered that empty divs are ok for a11y.

Reply:
We had an accessibility audit from Dublin where this was an issue (empty divs).

Essentially, the `aria-controls` ID on the `TitleButton` has to be exist in the html (at all times).

`aria-controls={collapsed-section-${uuid}}`

If we hide the box, then when the section isn't expanded we fail accessibility since an element with that ID doesn't exist in the html. For Dublin (for the Accordion component) we fixed that by making sure the Box element always existed.

Reply:
@amanda-anderson I see. So we're good then, right?